README.md

```diff
@@ -34,128 +37,125 @@ Als Partner können sowohl Personen als auch Organisationen angelegt werden. Die
-    "ort" : "..."
-  },
-  "avatarUrl": "https://www.europace2.de/partnermanagement/d71c58aeaed4022384b169f83e0d8842.avatar?anonymousAvatar=BENUTZER",
-  "bankverbindung" : {
-     "iban" : "...",
-     "bic" : "....",
-     "kontoinhaber": "...",
-     "referenzFeld" : "...."
-  },
-  "email" : "...",  
-  "externePartnerId" : "...", 		// Eine beliebige, extern erzeugte Id. Z.B. SAP oder CRM Nummer.
-  "faxnummer" : "...",
-  "firmenname" : "...",
-  "firmennameZusatz" : "...",
-  "fusszeileFuerAussenauftritt" : "...",        // mit \n  als Zeilentrenner
-  "geburtsdatum" : "1970-01-01",                // ISO-8601 Calender extended(YYYY-MM-DD) format.
-  "gesperrt": false,                            // default: false 
-  "gesperrtTransitiv": false,                   // true, wenn ein übergeordneter Partner gesperrt ist. (readonly)
-  "id":"...",                                   // EUROPACE 2 PartnerId (readonly)
-  "mobilnummer" : "...",
-  "nachname" : "...",
-  "rechtDarfEinstellungenOeffnen" : false,              //default false
-  "rechtDarfPartnerAnlegen" : false,                    //default false
-  "rechtDarfBaufiSmartEinstellungenVornehmen": false,   //default false
-  "rechtDarfBaufiSmartNutzen": true,                    //default true
-  "rechtDarfBaufiSmartVorgaengeUeberEuropaceOberflaecheAnlegen": true,      //default true
-  "rechtEchtgeschaeftErlaubt" : true,                   //default false
-  "rechtDarfErgebnisListeNutzen" : false,               //default false
-  "rechtDarfLoeschen": false,                           //default false
-  "rechtKreditSmartSichtbar": false,                    //default false
-  "rechtKreditSmartEchtgeschaeftErlaubt": false,        //default false
-  "rechtDarfKreditSmartVersicherungAnbieten" : true,    //default false
-  "rechtDarfKreditSmartVorgaengeUeberEuropaceOberflaecheAnlegen": true,     //default true
-  "titelFunktion" : "...",
-  "telefonnummer" : "...",
-  "typ": "PERSON",            				  
-  "vorname" : "...",
-  "webseiteUrl" : "...",
-  "aufsichtsBehoerde" : "...",
-  "registrierungsNummer" : "...",
-  "gehoertZuKreditinstitut" : "false"   // default false, markiert Personen, die Mitarbeiter eines Kreditinstitutes sind
+    "partnerId": "...",
+    "vorname": "...",
+    "nachname": "...",
+    "typ": "PERSON",      
+    "email": "...",
+    "avatar": "...",
+    "gesperrt": false,
+    "kreditsachbearbeiter": false,
+    "parent": {
+        "partnerId": "..."
+    },
+    "anrede": "HERR",     // alternativ: FRAU
+    "telefonnummer": "...",
+    "firmenname": "...",
+    "anschrift": {
+        "strasse": "...",
+        "hausnummer": "...",
+        "plz": "...",
+        "ort": "..."
+    },
+    "bankverbindung": {
+        "kontoinhaber": "...",
+        "bic": "...",
+        "iban": "..."
+    }
 }
 ```
 
 Dabei gilt:
 - Die Reihenfolge der Attribute spielt keine Rolle.
 - Die Attributbezeichner sind case-sensitive.
+- Zeilenumbrüche können durch '\n' erreicht werden.
```

Comment:
Wie ist denn das gemeint, bzw. worauf bezieht sich "Dabei gilt"? Den ganzen "Dabei gilt" Block würde ich komplett weg lassen. Wer baut denn heute noch JSON per Hand?

Reply:
Hmm, war in der alten Doku auch schon so enthalten und macht IMO auch Sinn. Kann man später mal weglassen. Die Änderung hier kommt durch dieses komsiche Fusszeilen-Feld, was wir in PEX 2.0 nicht haben. Musst daher den Verweise darauf rausenehmen.